app/controllers/hosts_controller.rb

```diff
@@ -936,6 +936,7 @@ def host_attributes_for_templates(host)
     # should be ignored by Host::Discovered or any other Host class that does
     # not have that attribute
     host_attributes = host.class.attribute_names
+    host_attributes << 'compute_attributes' if host.managed
```

Comment:
Just introduced this - let's see if it makes a difference (I was unable to reproduce the failures consistently)